dashboard/lti_new.py

```diff
@@ -13,7 +13,7 @@
 from pylti1p3.contrib.django import DjangoOIDCLogin, DjangoMessageLaunch, DjangoCacheDataStorage
 from pylti1p3.tool_config import ToolConfDict
 
-from dashboard.models import Course, CourseViewOption
+from dashboard.models import Course, CourseViewOption, User as mylaUser
```

Comment:
Can we alias instead as `MylaUser`, so we can tell it's a class?

Reply:
Is this just about case?  MylaUser vs mylaUser?

Reply:
Haha, um, yes, it is about case. The Python standard (which as you'll often hear is a recommendation, not a rule) is to use `CapWords` for classes (https://www.python.org/dev/peps/pep-0008/#class-names). I was/would be momentarily confused if a class didn't follow this naming scheme. However, if you feel strongly about wanting it to be `camelCase`, I will not fight you on it.

Reply:
Yeah I'd really just import this as the class it is `User` or minimally change it to `MylaUser`. It's just a case issue. 

We're actually branded as `MyLA` so technically should be `MyLAUser` but that is a nitpick.